src/ray/stats/metric_defs.cc

```diff
@@ -196,6 +185,43 @@ DEFINE_stats(gcs_storage_operation_count,
              (),
              ray::stats::COUNT);
 
+/// Object store
+/// Local object store memory of different types reported from multiple
+/// components. Reported metric from different subtypes, e.g. Location
+/// and Type could overlap. Reported metric within the same type might
+/// also overlap.
+DEFINE_stats(object_store_memory_bytes,
+             "Object store memory by various sub-kinds on this node",
+             /// Location:
+             ///    - InMemory: currently in memory (ObjectManager)
+             ///    TODO(rickyx): current spilled
+             ///    - Spilled: spilled to disk (LocalObjectManager)
+             ///    - Fallback: fallback allocated (ObjectManager)
+             /// Type:
+             ///     reported from LocalObjectManager
```

Comment:
I'm a little confused. These states are all for spilled objects only?

Reply:
Oh no, I think the relevant spilled objects are: 
- Spilled (current spilled size on external storage, WIP) 
- PendingRestore (the logical type, no physical memory back up as you mentioned in another comment)
- PendingSpill (still in memory)

The rest are not really about spilled objects.